Compatibility/runtimeconfig.template.json

```diff
@@ -0,0 +1,170 @@
+    //        // Default = True, AppContext switch can override it to False
+    //        ShouldRender = !CoreAppContextSwitches.ShouldNotRenderInNonInteractiveWindowStation
+    //      ELSE 
+    //        // Desktop/interactive mode, including WTSDisconnected scenarios
+    //        // Default = False, AppContext switch can override it to True
+    //        ShouldRender = CoreAppContextSwitches.ShouldRenderEvenWhenNoDisplayDevicesAreAvailable
+    //        END IF
+    //      END IF"
+    "Switch.System.Windows.Media.ShouldRenderEvenWhenNoDisplayDevicesAreAvailable": false,
+    "Switch.System.Windows.Media.ShouldNotRenderInNonInteractiveWindowStation": false,
+
+    // Reserved for future use
+    "Switch.System.Windows.DoNotUsePresentationDpiCapabilityTier3OrGreater": false,
+
+    // 
+    "Switch.System.Windows.AllowExternalProcessToBlockAccessToTemporaryFiles": false,
+
+    // Malicious managed objects could be placed in the clipboard lying about its format, 
+    // to fix this OleConverter now restricts object deserialization in some cases.
+    // When this switch is enabled behavior falls back to deserializing without restriction.
+    "Switch.System.Windows.EnableLegacyDangerousClipboardDeserializationMode": false,
+
+    // FrameworkAppContextSwitches
+
+    // Switch to enable non-adorner based rendering of TextSelection in TextBox and PasswordBox
+    "Switch.System.Windows.Controls.Text.UseAdornerForTextboxSelectionRendering": true,
+
+    // https://github.com/microsoft/dotnet/blob/master/Documentation/compatibility/wpf-layout-rounding-of-margins-has-changed.md
+    "Switch.MS.Internal.DoNotApplyLayoutRoundingToMarginsAndBorderThickness": false,
+
+    // https://github.com/microsoft/dotnet/blob/master/Documentation/compatibility/wpf-grid-allocation-of-space-to-star-columns.md
+    "Switch.System.Windows.Controls.Grid.StarDefinitionsCanExceedAvailableSpace": false,
+
+    // https://github.com/microsoft/dotnet/blob/master/Documentation/compatibility/wpf-tabcontrol-selectionchanged-and-selectedcontent.md
+    "Switch.System.Windows.Controls.TabControl.SelectionPropertiesCanLagBehindSelectionChangedEvent": false,
+
+    // https://github.com/microsoft/dotnet/blob/master/Documentation/compatibility/wpf-MasterDetail-ADOdata-PrimaryKey.md
+    "Switch.System.Windows.Data.DoNotUseFollowParentWhenBindingToADODataRelation": false,
+
+    // Switch to enable IList indexer hiding a custom indexer in a binding path
+    "Switch.System.Windows.Data.Binding.IListIndexerHidesCustomIndexer": false,
+
+    // Switch to enable appending the local assembly version to the Uri being set 
+    // for ResourceDictionary.Source via Baml2006ReaderInternal.
+    "Switch.System.Windows.Baml2006.AppendLocalAssemblyVersionForSourceUri": false,
+
+    // Switch to enable keyboard navigation from a hyperlink to go to the wrong place
+    "Switch.System.Windows.Controls.KeyboardNavigationFromHyperlinkInItemsControlIsNotRelativeToFocusedElement": false,
```

Comment:
Might want compat doc links (if available) for most of these now that I think about it.